playbooks/rhbk.yml

```diff
@@ -40,11 +40,63 @@
           redhat.rhbk collection v{{ galaxy_version | default('0.0.0-dev') }} is for [Technical Preview](https://access.redhat.com/support/offerings/techpreview).
           If you have any issues or questions related to collection, please don't hesitate to contact us on <Ansible-middleware-core@redhat.com> or open an issue on
           <https://github.com/ansible-middleware/keycloak/issues> referring to the keycloak_quarkus role
+      - placeholder: rhbk_playbook
+        content: |
+          * The below example playbook can install based on the defined variables (using most defaults).
+          
+          ```
+          ---
+          - name: Playbook for rhbk Hosts
+            hosts: all
+            vars:
+              rhbk_admin_password: "remembertochangeme"
+            roles:
+              - redhat.rhbk.rhbk
+          ```
+          
+          Both playbooks include the `rhbk` role, with different settings, as described in the following sections.
+
+          For full service configuration details, refer to the [rhbk role README](https://github.com/ansible-middleware/keycloak/blob/main/roles/keycloak/README.md).
```

Comment:
https://github.com/ansible-middleware/keycloak/blob/main/roles/keycloak/README.md
should actually point to
https://github.com/ansible-middleware/keycloak/blob/main/roles/keycloak_quarkus/README.md
since downstream rhbk takes keycloak_quarkus as the rhbk role

Reply:
Thanks! Did the changes.